format/FormatCBFFull.py

```diff
@@ -7,9 +7,8 @@
 from __future__ import absolute_import, division, print_function
 
 import sys
-from builtins import range
```

Comment:
Maybe slightly OCD but I would probably have split that into a second commit i.e. one commit for numpy->np and a second for "misc housekeeping since I touched things"

But I can see here that these are flake8 warnings probably so I do not feel at all strongly

Reply:
split up commits